civihr_default_theme/scss/mail.scss

```diff
@@ -55,19 +55,28 @@ table.social {
 
 }
 .social .soc-btn {
```

Comment:
I understand the need for `.soc-td` (since you're going to wrap the `<a>` into a `<td>`), but what are these changes for `.soc-btn` for?